include/zephyr/drivers/can.h

```diff
@@ -30,6 +30,8 @@ extern "C" {
 /**
  * @brief CAN Interface
  * @defgroup can_interface CAN Interface
+ * @since 1.14
```

Comment:
I thought I added CAN in 1.13 🤔

Reply:
I just got it from here:
https://github.com/zephyrproject-rtos/zephyr/blob/main/doc/develop/api/overview.rst?plain=1#L62

:eyes:

Reply:
Whoever added that entry had it wrong. The initial CAN driver API was added in 9e12807a6b59816ad0f86e7f4b3b382a02c767f7, which is present even in v1.12.